saskatoon/saskatoon/settings.py

```diff
@@ -225,7 +225,8 @@ def get_drf_template_mode() -> str:
    'DEFAULT_RENDERER_CLASSES': (
         get_drf_template_mode(),
```

Comment:
@mina-andajos if I set "DRF_BROWSABLE_API_MODE=yes" in .env I get `Object of type HarvestFilterForm is not JSON serializable` when trying to GET the harvest page. Is this expected behavior? If not should we drop drf browser mode?

Reply:
it's not no
I think it just needs a quick fix with the serializer. I will check it out.

Reply:
the forms that we pass to the context aren't serializeable. I remember discussing this before. I could add an extra condition that if the template renderer is the drf one then we could pass just the json response without a custom context

Reply:
something like this:
```py
    def list(self, request, *args, **kwargs):
        self.template_name = 'app/list_views/harvest/view.html'
        self.serializer_class = HarvestListSerializer

        response = super(HarvestViewset, self).list(request, *args, **kwargs)
        if request.accepted_renderer.format in ('json', 'api'):
            return Response(response.data)
```